src/test/java/org/folio/permstest/RestVerticleTest.java

```diff
@@ -44,6 +44,7 @@
 import org.junit.After;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.Timeout;
```

Comment:
`io.vertx.ext.unit.junit.Timeout` will allow `Test(timeout = ...)` to overwrite the `@Rule`.

Reply:
ok